modules/cli/src/main/scala/scala/cli/commands/pgp/PgpCreate.scala

```diff
@@ -0,0 +1,16 @@
+package scala.cli.commands.pgp
+
+import caseapp.core.RemainingArgs
+
+import scala.cli.commands.ScalaCommand
+import scala.cli.signing.commands.{PgpCreate => OriginalPgpCreate, PgpCreateOptions}
+
+object PgpCreate extends ScalaCommand[PgpCreateOptions] {
```

Comment:
Does `ScalaCommand` refers to the fact that it is written in Scala?

Reply:
It stands for Scala CLI command, a command of Scala CLI. It adds shared settings to `caseapp.core.app.Command`.